### ISSUE TITLE
[styled-components] Comments following interpolations are duplicated

### ISSUE THREAD

**Issue by jacobwgillespie** [prettier/prettier#5381]
If a standalone Styled Components interpolation is following by a CSS comment, the CSS comment is duplicated in the resulting output.

**Prettier 1.15.1**

[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEBlGBPAGzgBMBhCAWwAdoEsBeTDQkgOmIEsA3AAwB0omTABJgAQygBJWHABONAmJgdoAXwEChAegBUmMZkgUKdTDq0bBmVmIBKAVyKZgmoYYgEIspJlklX6lA8IAA0IBBUyujIoGKyshAA7gAKcQhoyCBiXBAcxKEgAEayYmAA1nAw2FSlHFAA5sgwsvZwYQAWMBQEAOrtHPBoNWBw2OkD3AN4mWBoGWF1aHIwySX1FGLIAGZiBEthAFZoAB4AQiXlldhiJgAydXDbu-sgR8fYdfVEAIr2EPBPPZtEA1WRLWSZQpiQqEaAFKiyOowHp5GDtZAADgADGEERAlj0SlRMgi4OCuI8wn4AI72Dh+VZidabJA7IFhJYUDhNFrAtCfH5-AGs57AmDQlHENHIABMYWaYg4BE+5GMLJAUFoBXsSwAKtCMiKgapVEA)

```sh
# Options (if any): none / defaults
```

**Input:**
```jsx
const StyledComponent = styled.div`
  ${anInterpolation}

  /* a comment */

  .aRule {
    color: red
  }
`;
```

**Output:**
```jsx
const StyledComponent = styled.div`
  ${anInterpolation}

  /* a comment */

  /* a comment */

  .aRule {
    color: red;
  }
`;
```

Each subsequent format will continue to duplicate all comments, for example formatting that output again results in the following ([playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEBlGBPAGzgBMBhCAWwAdoEsBeTDQkgOmIEsA3AAwB0omTABJgAQygBJWHABONAmJgdoAXwEChAegBUmMZkgUKdTDq0bBmXfsOUTsMxaibMrMQCUArkUzBXQpAEELJImLIkANyu6lA80VAgADQgEFTK6MigYrKyEADuAAo5CGjIIGJcEBzEySAARrJiYADWcDDYVM0cUADmyDCyXnApABYwFAQA6qMc8GhdYHDYpXPcc3jlYGhlKT1ocjCFTb0UYsgAZmIEBykAVmgAHgBCTa3t2GImADI9cJfXW4gB6PbA9XpEACKXgg8ABNxGIC6sgOsnK9TE9UI0DqVFkPRgUxqMFGyAAHAAGFJ4iAHKZNKjlPFwVFcf4pCIARy8HAixzEp3OSCuCJSBwoHAGQ0RaHBUJhcOFgMRMExROIJOQACYUoMxBwCODyMYhSAoLQ6l4DgAVTFlJUI1SqIA)):

```jsx
const StyledComponent = styled.div`
  ${anInterpolation}

  /* a comment */

  /* a comment */

  /* a comment */

  /* a comment */

  .aRule {
    color: red;
  }
`;
```

**Expected behavior:**

```jsx
const StyledComponent = styled.div`
  ${anInterpolation}

  /* a comment */

  .aRule {
    color: red;
  }
`;
```

---

Adding a semicolon to the end of the interpolation results in the expected behavior:

```diff
-  ${anInterpolation}
+  ${anInterpolation};
```

However I don't believe a semicolon should be required in this case, given that the interpolation could be the result of a call to `css` or other similar multi-rule mixins.

**Comment by Kovensky** [comment]
I was about to report this myself. https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBnGBPANnAJgOlwEsA3AAwB0oACagEmFQgFs4BhVVAXyqtoHoAVNQCC1SM1axqg-n2r4mrAMpwcYGBABO1YDyhkQAGhAQADjCLRUyUAEMtWiAHcACg4Q2UdkhCK5jEAAjLTswAGs4GGUzMKIoAHNkGC0AVzgTAAsYZiwAdUyieFRYsDhVKFQi0iKMZHBOQPjUOC0YV1CE5jtkADM7LBaTACtUAA8AIVCIqOU7VgAZeLg+gaGQUbHleIScAEVUiHhVwYyQWK0WrXqguyDsaECzLXiYPP8YTOQADgAGE2eEBaeVCZnqzzgVxIKxMWjgAEdUkQ4R07F0ekh+qcTC1mERkmkzlVEvtDsdMWszjA7u9cJ9kAAmEwpOxELA7Ngsbr1KDQGEgVItAAqdy8WJaXC4QA

The comment need not precede a rule; as long as it is _between_ an interpolation and any other non-whitespace node it will get duplicated on every run.

**Comment by ikatyang** [comment]
The fix is available in [`1.15.2`](https://github.com/prettier/prettier/blob/master/CHANGELOG.md#1152).
